{
  "contract": "mishandled",
  "tool": "mythril",
  "start": 1640178635.2058876,
  "end": 1640178638.7686985,
  "duration": 3.5628108978271484,
  "analysis": {
    "error": null,
    "issues": [
      {
        "address": 276,
        "code": "msg.sender.send(amountToWithdraw)",
        "debug": "SOLVER OUTPUT:\ncalldata_SendBack_0: 0x5fd8c71000000000000000000000000000000000000000000000000000000000\ncalldatasize_SendBack: 0x4\ncallvalue: 0x0\n",
        "description": "In the function `withdrawBalance()` a non-zero amount of Ether is sent to msg.sender.\nIt seems that this function can be called without restrictions.",
        "filename": "/data/demo_analysis/source_code/mishandled.sol",
        "function": "withdrawBalance()",
        "lineno": 14,
        "title": "Ether send",
        "type": "Warning"
      },
      {
        "address": 276,
        "code": "msg.sender.send(amountToWithdraw)",
        "debug": "",
        "description": "The return value of an external call is not checked. Note that execution continue even if the called contract throws.",
        "filename": "/data/demo_analysis/source_code/mishandled.sol",
        "function": "withdrawBalance()",
        "lineno": 14,
        "title": "Unchecked CALL return value",
        "type": "Informational"
      }
    ],
    "success": true
  }
}